test/parallel/test-assert-fail.js

```diff
@@ -1,53 +1,76 @@
 'use strict';
+
 const common = require('../common');
 const assert = require('assert');
 
-// no args
+// No args
 assert.throws(
   () => { assert.fail(); },
   common.expectsError({
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
-    message: 'undefined undefined undefined'
+    message: 'Failed',
```

Comment:
hmmmm.... right.
This is https://github.com/nodejs/node/pull/13974 leaking into my backport. I can take it out.